### PR TITLE
GitHub Label Management

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to update the set of tools and data associated with managing GitHub labels across the set of Azure SDK repositories.

The previous set of Python scripts have been replaced by Alex Ghiondea's [GHCreator](https://github.com/AlexGhiondea/ghcreate) tool with a set of PowerShell wrappers.  Label and repository data are the result of recent leadership efforts to normalize labels and reflect the currently agreed upon common set.

# Considerations

I decided against managing the GH Creator tool as a submodule, as that tends to add complexity and is unfamiliar to many.  Since the tool is stable for what is needed at present and is not rapidly evolving, I concluded a binary drop would be the better way to go.

### CODE REVIEW

File: tools/github-labels/repositories.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Azure\azure-sdk-for-java-pr
+Azure\azure-sdk-for-python
+Azure\azure-sdk-for-python-pr
+Azure\azure-sdk-for-js
+Azure\azure-sdk-for-js-pr
+Azure\azure-sdk-for-c
+Azure\azure-sdk-for-c-pr
+Azure\azure-sdk-for-cpp
+Azure\azure-sdk-for-go
+Azure\azure-sdk-for-go-pr
```
Comment (benbp) [inline tools/github-labels/repositories.txt:13]: no android/ios?

Comment (jsquire) [reply tools/github-labels/repositories.txt:13]: I can't speak to why, but it wasn't part of these efforts.

**Comment by jsquire** [comment]
./check-enforcer override

File: tools/github-labels/repositories.txt (hunk 1 above)
Comment (benbp) [reply tools/github-labels/repositories.txt:13]: I figured that was probably the case, just wanted to double check.

**Pull request merged**